### PR TITLE
[RCIAM-138] Enlist and save authenticating authority attribute during user registration

### PR DESCRIPTION
**Pull request by ioigoume**
No database change is required.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Retrieve AuthenticatingAuthority and depict in CO Person's canvas/profile
 - IdP hinting for RCAUTH plugin
 - Add support for hiding attributes from the enrollment form. Admins can hide an enrollment flow attribute by setting the value of Hidden from enrollment form view to true in the Enrollment Attribute configuration page. By default the value is false/empty and all the configured Enrolment Attributes will be displayed in the enrollment form.
-
+- Retrieve AuthenticatingAuthority during user restistration
```
Comment (NicolasLiampotis) [inline CHANGELOG.md]: Typo in registration

File: app/Model/CmpEnrollmentAttribute.php [hunk 2]
```diff
@@ -131,6 +131,13 @@ public function availableAttributes() {
         'ldap_name' => 'edu_person_affiliation',
         'saml_name' => 'edu_person_affiliation'
       ),
+      'authn_authority' => array(
+        'required'  => RequiredEnum::Optional,
+        'label'     => _txt('fd.authn_authority'),
+        'env_name'  => 'CMP_EF_AUTHNAUTHORITY',
+        'ldap_name' => 'AuthenticatingAuthority',
```
Comment (NicolasLiampotis) [inline app/Model/CmpEnrollmentAttribute.php]: If `AuthenticatingAuthority` is not part of an ldap schema then `ldap_name` should be set `''`

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ioigoume) [reply CHANGELOG.md]: Done.

File: app/Model/CmpEnrollmentAttribute.php (hunk 2 above)
Comment (ioigoume) [reply app/Model/CmpEnrollmentAttribute.php]: Replacing with an empty string value

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**